frontend/package.json

```diff
@@ -46,7 +46,10 @@
     "react-dom": "^18.2.0",
     "react-loading-skeleton": "^3.4.0",
     "react-tailwindcss-datepicker": "^1.6.6",
-    "tiptap": "^1.32.2"
+    "sharp": "^0.33.4",
```

Comment:
why are we bringing in sharp? and yarn?

Reply:
ah shit think that was a grief on my part and i forgot to remove it

Reply:
its already gone
think this is old code

Reply:
its not.. this is the diff on your most recent commit... please dont resolve threads, unless its solved, reduces visibility. If the diff shows its being added in, it means its not removed